src/CommunityVoices/App/Website/View/Image.php

```diff
@@ -273,63 +241,6 @@ public function postImageUpload($request)
 
     public function getImageUpdate($request)
     {
-        $paramXML = new Helper\SimpleXMLElementExtension('<form/>');
```

Comment:
Please entirely remove this route if you seek to take this route.

Reply:
Assuming that you mean the function itself? And on routes.json?

Reply:
Yes.  I don't see a large reason for the route existing if it will not return a response and therefore will just be a 500 error.